src/Relations/EmbedsOneOrMany.php

```diff
@@ -42,8 +43,8 @@ abstract class EmbedsOneOrMany extends Relation
      */
     public function __construct(Builder $query, Model $parent, Model $related, string $localKey, string $foreignKey, string $relation)
     {
-        $this->query      = $query;
-        $this->parent     = $parent;
+        parent::__construct($query, $parent);
+
         $this->related    = $related;
```

Comment:
This property is also being set in the parent constructor, but when I tried to change it a few mocks broke and I'm not sure how to update those. So I left it alone 😅